lib/adapters/websockex.ex

```diff
@@ -0,0 +1,90 @@
+if Code.ensure_loaded?(WebSockex) do
+  defmodule Janus.Transport.WS.Adapters.WebSockex do
+    @moduledoc """
+    Adapter for [WebSockex](https://github.com/Azolo/websockex).
+    """
+
+    use Janus.Transport.WS.Adapter
+    use WebSockex
+
+    @impl true
```

Comment:
You're implementing 2 behaviours, so you should rather use `@impl Adapter` and `@impl WebSockex`